qa/automation/CareerPortal.Tests.Scenarios/Savostin/CV_sending.feature

```diff
@@ -0,0 +1,179 @@
+	Then form is not sent
+	And following are displayed appropriate fields:
+	|"invalid name format"                                |
+	|"invalid email format"                               |
+	|"attach CV" becomes red                              |
+	|"I agree to the personal data processing" becomes red|
+
+Scenario Outline:1.0.0_02 form with only invalid name on it can not be sent
+	When user clicks on "Send" button
+	And all fields are empty except name with <name>
+	Then form should not be sent
+	And following errors should be displayed appropriate fields:
+	|"invalid name format"                                |
+	|"this field is required"                             |
+	|"attach CV" becomes red                              |
+	|"I agree to the personal data processing" becomes red|
+
+	Examples: 
+	|name            |
+	|�������         |
+	|������� ��������|
+	|Eugene          |
+	|asdadgsgdfm     |
+	|@!#!@#$!@$      |
+
+Scenario Outline:1.0.0_03 form with only valid name on it can not be sent
+	When user clicks on "Send" button
+	And all fields are empty except name with <name>
+	Then form should not be sent
+	And following errors should be displayed appropriate fields:
+	|"this field is required"                             |
+	|"attach CV" becomes red                              |
+	|"I agree to the personal data processing" becomes red|
+
+	Example: Eugene Savostin
+
+Scenario Outline:1.0.0_04 form with only invalid email on it can not be sent
+	When user clicks on "Send" button
+	And all fields are empty except email with <email>
+	Then form should not be sent
+	And following errors should be displayed appropriate fields:
+	|"this field is required"                             |
+	|"invalid email format"                               |
+	|"attach CV" becomes red                              |
+	|"I agree to the personal data processing" becomes red|
+
+	Examples: 
+	|email      |
+	|dadassdf   |
+	|dadada@mail|
+
+Scenario Outline:1.0.0_05 form with only valid email on it can not be sent
+	When user clicks on "Send" button
+	And all fields are empty except email with <email>
+	Then form should not be sent
+	And following errors should be displayed appropriate fields:
+	|"this field is required"                             |
+	|"attach CV" becomes red                              |
+	|"I agree to the personal data processing" becomes red|
+
+	Examples: 
+	|email            |
+	|myemail@gmail.com|
+	|test@mail.ru     |
+
+Scenario:1.0.0_06 form with only attached CV on it can not be sent
+	When user clicks on "Send" button
+	And all fields are empty except CV "valid_file_to_attach.docx" is attached
+	Then form should not be sent
+	And following errors should be displayed appropriate fields:
+	|"this field is required"|
+	|"this field is required"|
+	|"I agree to the personal data processing" becomes red|
+
+Scenario Outline:1.0.0_07 form with all valid data except name on it can not be sent
+	Given field name contains <name>
```

Comment:
Same here: As you have only one example of the email in this scenario outline, I believe it would be better to change it to regular scenario and just add the name somehow so:

And field email contains " eugene@gmail.com "
...

Otherwise, you can add some other examples of the names and emails.